packages/core-v2/src/visual-model/visual-entity.ts

```diff
@@ -45,6 +46,21 @@ export interface VisualNode extends VisualEntity {
 
 }
 
+/**
+ * Represents a visual node, which represents {@link representedVisualModel}.
+ */
+export interface VisualDiagramNode extends VisualEntity {
```

Comment:
@jakubklimek  @RadStr So the intention is that we store label and description in the visual model for digram node? We do not do this for other types of nodes, instead we load the value from the semantic model. 
Why is this different?

Reply:
@skodapetr well... this node does not have a semantic source. It is purely visual. So that could be one reason